### ISSUE TITLE
[ESP32 bridge-app] Endpoint 0 not found in IncreaseClusterDataVersion!

### ISSUE THREAD

**Issue by VOMediola** [project-chip/connectedhomeip#14948]
Latest build throws following errors on start

```
I (775) chip[DL]: Device[Light 1]: ONLINE
E (775) chip[DMG]: Endpoint 0, Cluster 0x0000_0039 not found in IncreaseClusterDataVersion!
I (775) chip[DL]: Device[Light 2]: ONLINE
E (775) chip[DMG]: Endpoint 0, Cluster 0x0000_0039 not found in IncreaseClusterDataVersion!
I (785) chip[DL]: Device[Light 3]: ONLINE
E (795) chip[DMG]: Endpoint 0, Cluster 0x0000_0039 not found in IncreaseClusterDataVersion!
I (805) chip[DL]: Device[Light 4]: ONLINE
E (805) chip[DMG]: Endpoint 0, Cluster 0x0000_0039 not found in IncreaseClusterDataVersion!
```
Guess it has something to do with https://github.com/project-chip/connectedhomeip/pull/14471 only how to add DataVersion to Endpoint 0, which is not dynamic and is a "Bridge" cluster itself?

**Comment by PSONALl** [comment]
@VOMediola this issue is solved on the latest CHIP HEAD d671a523745079b1ea1778ee4885159868d9c002

**Comment by dhrishi** [comment]
@VOMediola Can you please confirm and close this issue if it is working as expected.
